test/README.md

```diff
@@ -0,0 +1,35 @@
+# Sample Test Setup
+
+### Required files
+
+Add files with the following information to the `SIS_TEST_DIR` (by default, this
+is the current directory).
+
+1. `${SIS_TEST_DIR}/.usernames.json`:
```

Comment:
@davidlimcheng I think it's a hack with Github Markdown. If you make a numbered list that begins with 1. it will simply order them properly, that way you don't have to re-number the items in the list.

```
1. Black cat
1. Blue bird
1. Green snake
1. Red dog
```

Is rendered as:

1. Black cat
1. Blue bird
1. Green snake
1. Red dog